joomla/joomla-cms
# [feature] - Batch process to add URLs in Redirect component

With this PR a new feature is introduced to the Redirect component: 

Batch process to add any number of URLs in one step. The URLs can be added in a modal box which is opened after a click on the new batch button. To speed up the adding, not only the old URL but also the new URL can be added in the same process. The URLs have to be separated with a | (pipe symbol):

`http://example.com/old|http://example.com/new`

The old URL is mandatory, the new URL not.The host is added automatically to the old URL if not provided which is necessary for the redirect component to work properly.

If an entry already exists, an error "Duplicate entry" is thrown.

**How to test**
- Add the PR with the help of the Patchtester component
- Go to "Components" - "Redirect"
- Click on the "Batch" button
- Enter some URLs and click on the "Process" button
- Entered URLs should be displayed in the overview list
